api/fasit.go

```diff
@@ -110,7 +111,7 @@ type NaisResource struct {
 	propertyMap  map[string]string
 	secret       map[string]string
 	certificates map[string][]byte
-	ingresses    map[string]string
+	ingresses    []Ingress
```

Comment:
I dont think we should depend on manifest types here. Create a fasit type

Reply:
Thanks. This was unclear to me, especially since the entire data model resides under a single `api` module.